app/views/studies/export_search_results.xlsx.axlsx

```diff
@@ -0,0 +1,15 @@
+wb = xlsx_package.workbook
+wb.add_worksheet(:name => "Studies") do |sheet|
+  sheet.add_row ["nct_id", "rating", "title", "status", "started", "completed"]
+  @studies.each { |study|
+    print study[:nct_id]
```

Comment:
Please remove this print statement -- I'm not sure this is something we want up to 1000 of in our logs